src/maxdiffusion/configs/base_flux_dev.yml

```diff
@@ -54,7 +54,7 @@ precision: "DEFAULT"
 # Set true to load weights from pytorch
 from_pt: True
 split_head_dim: True
-attention: 'flash' # Supported attention: dot_product, flash
+attention: 'cudnn_flash_te' # Supported attention: dot_product, flash, cudnn_flash_te
```

Comment:
let's make flash the default and add a section in the README on how to use with GPUs.